src/main/java/eod/Game.java

```diff
@@ -30,12 +30,24 @@ public void start() {
         A.drawFromDeck(3);
         B.drawFromDeck(3);
 
-        while(handIsInvalid(A) || handIsInvalid(B)) {
-            if(handIsInvalid(playerOrder[0])) {
-                Player first = playerOrder[0];
-
+        boolean hasInvalid;
+        int drawRound = 0;
+        do {
+            hasInvalid = false;
+            for(int i = 0;i < playerOrder.length;i++) {
+                Player player = playerOrder[i];
+                Player other = playerOrder[i==0?1:0];
+
+                if(handIsInvalid(player)) {
+                    hasInvalid = true;
+                    other.drawFromDeck(1);
+                    player.dropCards();
+                }
```

Comment:
Bug: player won't have any cards after its first drop

Reply:
Fixed in the next commit